src/components/ParticipantList.jsx

```diff
@@ -54,11 +59,51 @@ export default function ParticipantList({ studyID, title, onParticipantSelect, s
         })()
     }, [participants])
 
+
     let addParticipant = async () => {
         let newCount = state.newCount
         let ids = []
-        for (let i = 0; i < newCount; i ++)
-            ids = [...ids, (await LAMP.Participant.create(studyID, { study_code: '001' })).data]
+        let id = []
+        let tempPassword = []
+
+        for (let i = 0; i < newCount; i ++) {
+            id = (await LAMP.Participant.create(studyID, { study_code: '001' })).data
+            tempPassword = id
+            if (!!(await LAMP.Credential.create(id, tempPassword, 'Temporary Login')).error) {
+                enqueueSnackbar(`Could not create credential for ${id}.`, { variant: 'error' , anchorOrigin: {
+                    vertical: 'bottom',
+                    horizontal: 'right',
+                }})
+            } else {
+                enqueueSnackbar(`Successfully created Participant ${id}. Tap the expand icon on the right to see credentials and details.`, {
+                    variant: 'success', 
+                    persist: true, 
+                    content: (key, message) => 
+                        <SnackMessage id={key} message={message}>
+                            <TextField variant="outlined" size="small" label="Temporary email address" value={`${id}@lamp.com`} />
+                            <div style={{ height: 16 }} />
+                            <TextField variant="outlined" size="small" label="Temporary password" value={`${tempPassword}`} />
+                            <Grid item>
+                                <TextField 
+                                    fullWidth
+                                    label = 'One-time login link'
+                                    style={{ marginTop: 16 }}
+                                    variant="outlined"
+                                    value={_qrLink()}
+                                    onChange={event => {}}
+                                />
+                                <Tooltip title="Scan this QR code on a mobile device to automatically open a patient dashboard.">
+                                    <Grid container justify="center" style={{ padding: 16 }}>
+                                    <QRCode size={256} level="H" value={_qrLink()} />
```

Comment:
Change how you call the `_qrLink()` function here to: `_qrLink(email, password)` and substitute the email/password variables for the ones you used above.

Reply:
Change made.